packages/framework/src/Model/Customer/CustomerFacade.php

```diff
@@ -130,7 +130,9 @@ public function create(CustomerData $customerData)
         $this->em->persist($user);
         $toFlush[] = $user;
 
-        $this->em->flush($toFlush);
+        if (count($toFlush) > 0) {
```

Comment:
It will always be true here.

Reply:
These should be reported by PHPStan.
Is it broken?

Reply:
Hi Petr, modified. 

Hi Tomas, I'm not sure but there is default setting with allowed reports only for two kinds of warnings - CommentedOutCodeSniff and AssignmentInConditionSniff. And UnconditionalIfStatementSniff is only a warning but seems not allowed for reporting. Could you confirm this for me?